config/config.go

```diff
@@ -12,4 +12,5 @@ type Config struct {
 	GRPCAddress             string
 	GRPCSecure              bool
 	AgentAllowedStartupTime time.Duration
+	FilterLabels            string
```

Comment:
Just noticed this has to be set in main.go

Reply:
Good catch, fixed.